app/models/card.rb

```diff
@@ -1,2 +1,18 @@
 class Card < ActiveRecord::Base
+  validates :original_text, :translated_text, :review_date, presence: { message: I18n.t('error.validation.messages.cant_be_blank') }
```

Comment:
Line is too long. [132/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.